src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./app";
+import Firebase, { FirebaseContext } from './components/Firebase';
 import * as serviceWorker from "./serviceWorker";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <FirebaseContext.Provider value={new Firebase()}>
```

Comment:
You shouldn't pass `new Firebase()` into the provider. Read more about why here: https://reactjs.org/docs/context.html#caveats

You have a few options. You could instead create `new Firebase()` higher up, e.g
```javascript
const fb = new Firebase();
ReactDOM.render(
  ...
 <FirebaseContext.Provider value={fb}/>
  ...
)
```

Alternatively, you should consider not even having this as a something that passes through your React tree because your firebase itself isn't impacting the ui state but providing the data for the ui state. You can make your fb instance as a singleton, export that, and then wherever you need it, you can simply import it: https://www.sitepoint.com/javascript-design-patterns-singleton/